08LessonEight/homework/exercise-06.html

```diff
@@ -3,8 +3,35 @@
 
 <head>
     <title>Exercise 06</title>
+    <link rel="stylesheet" href="css/exercise-06-style.css" media="screen" title="no title" charset="utf-8">
 </head>
 
-<body></body>
+<body>
+<div class="pictures">
+  <img src="img/truck.jpg" alt="" />

```

Comment:
A very useful tip, if in your CSS you add `img { max-width: 100%; height: auto }` you will make the images responsive. See this fiddle. http://bit.ly/2bB0EPZ